server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileService.java

```diff
@@ -87,6 +87,7 @@ public BulkChangeResult bulkDeactivate(RuleQuery ruleQuery, String profile) {
   }
 
   private void verifyAdminPermission() {
+    // FIXME check for the permission of the appropriate organization, not the default one
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)